sites-global/global.json

```diff
@@ -0,0 +1,3 @@
+{
+	"name": "Answers Hitchhiker Theme"
```

Comment:
how does this file get use? does SGS merge this to another configuration?

Reply:
I'm not exactly sure how it's used. I saw it in other repos on Yext Sites so I figured it wouldn't hurt to add, but I can look more into it